index.html

```diff
@@ -0,0 +1,135 @@
+              <h3 class="program-tittle">Forum</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit.</p>
+            </div>
+            <div class="program ">
+              <img class="program-icon" src="images/program_icon_04.png" alt="">
+              <h3 class="program-tittle">Workshop</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit.</p>
+            </div>
+            <div class="program ">
+              <img class="program-icon" src="images/program_icon_05.png" alt="">
+              <h3 class="program-tittle">CC Ignite</h3>
+              <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum.</p>
+            </div>
+          </div>
+          <div class="join-culture" id="join">
+            <a class="joinnew" href="#">SEE THE WHOLE PROGRAM</a>
+          </div>
+        </section>
+        <section class="featured-activities" id="news">
+            
+        </section>
+        <section class="partners" id="sponsor" >
+          <h2 class="secondary-tittle partner-tittle" >Partner</h2>
+          <div class="bar-icon"></div>
+          <div class="partners-wrap">
+            <ul class="partners-list">
+              <li class="partner">
+                <a href="#"><img src="images/mozilla.png" alt="mozzila logo" width="150" height="50"></a>
+              </li>
+              <li class="partner1">
+                <a href="#"><img src="images/googlekorea.png" alt="google logo" width="150" height="50"></a>
+              </li>
+              <li class="partner1">
+                <a href="#"><img src="images/naver.png" alt="naver logo" width="130" height="50"></a>
+              </li>
+              <li class="partner1">
+                <a href="#"><img src="images/kakao.png" alt="Kakao logo" width="120" height="50"></a>
+              </li>
+              <li class="partner1">
+                <a href="#"><img src="images/airbnb.png" alt="Airbnb logo" width="130" height="50"></a>
+              </li>
+            </ul>
+          </div>
+        </section>
+      </main>
+      <footer class="main-footer">
+        <div class="footer-wrap">
+          <div class="footer-wrap">
+            <img class="footer-logo-main" src="images/logo.png" alt="logo creative commons korea" width="208" height="35">
+            <div class="footer-text-wrap">
+              <p class="footer-text">Copyright © 2015-2021 All Conference Alert. All Rights Reserved. worldconferencealerts.com</p>
+              <p class="footer-text-2">Lorem ipsum, dolor sit amet, explicabo quaerat. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptas, odit consequatur quae, a corrupti repudiandae impedit similique dignissimos.</p>
+            </div>
+          </div>
+        </div>
+      </footer>
```

Comment:
Kindly note that the mobile version of the homepage does not have a footer, it ends with the `more` button. Please check and update your design accordingly. :pray: 


 Your design            |  The  design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-30 07-44-37](https://user-images.githubusercontent.com/65233860/204736713-58b35311-4d73-4d0a-b85e-8cd448f8b79d.png)  |  ![Screenshot from 2022-11-30 07-44-55](https://user-images.githubusercontent.com/65233860/204736730-fc7f6b56-9155-41cf-93d6-5d67f8601f8e.png)